### PR TITLE
Modify save_quant_model to support different input and output filenames

### PR DESCRIPTION
**Pull request by wozna**
### PR types

New features
### PR changes

Others 
### Describe

This PR adds additional parameters to save_quant_model.py to define the names of the input and output files. The default version of the input and output file names remains the same as before.

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/tests/save_quant_model.py [hunk 1]
```diff
@@ -109,4 +150,6 @@ def transform_and_save_int8_model(original_path,
     test_args, remaining_args = parse_args()
     transform_and_save_int8_model(
         test_args.quant_model_path, test_args.int8_model_save_path,
-        test_args.ops_to_quantize, test_args.op_ids_to_skip, test_args.debug)
+        test_args.ops_to_quantize, test_args.op_ids_to_skip, test_args.debug,
+        test_args.save_model_filename, test_args.save_params_filename,
+        test_args.quant_model_filename, test_args.quant_params_filename)
```
Comment (sfraczek) [inline python/paddle/fluid/contrib/slim/tests/save_quant_model.py]: quant_xxx_filename should be first

Comment (wozna) [reply python/paddle/fluid/contrib/slim/tests/save_quant_model.py]: You are totally right. Thank you

**Review by sfraczek — APPROVED** [review]
LGTM!

**Review by lidanqing-vv — APPROVED** [review]
LGTM

**Pull request merged**